app/models/post.rb

```diff
@@ -22,4 +24,29 @@ def tweet_content
     max_title_length = 140 - url.length
     title[0...max_title_length] + url
   end
+
+  def tags_description
+    tags.map{ |tag| tag.title }.join(", ")
```

Comment:
Space missing to the left of {.<br>Pass `&:title` as an argument to `map` instead of a block.